internal/exec/process.go

```diff
@@ -223,24 +225,25 @@ func startProcess(argv0 string, argv []string, attr *ProcAttr) (pid int, handle
 		}
 	}
 
-	si := new(syscall.StartupInfo)
-	si.Cb = uint32(unsafe.Sizeof(*si))
-	si.Flags = syscall.STARTF_USESTDHANDLES
+	si := new(winapi.STARTUPINFOEX)
+	si.StartupInfo.Cb = uint32(unsafe.Sizeof(*si))
+	si.StartupInfo.Flags = syscall.STARTF_USESTDHANDLES
 	if sys.HideWindow {
-		si.Flags |= syscall.STARTF_USESHOWWINDOW
-		si.ShowWindow = syscall.SW_HIDE
+		si.StartupInfo.Flags |= syscall.STARTF_USESHOWWINDOW
+		si.StartupInfo.ShowWindow = syscall.SW_HIDE
 	}
 	si.StdInput = fd[0]
 	si.StdOutput = fd[1]
 	si.StdErr = fd[2]
+	si.LpAttributeList = sys.ProcThreadAttrList
```

Comment:
How is this value used?

Reply:
It's not used anywhere by us, its passed to the CreateProcess call. Making of an attribute list can be seen in the unit tests.

Reply:
but this value isn't used in this function or given to any other here right?

Reply:
No, I assume some pointer tricks are done to access the attribute list if the EXTENDED_STARTUP_INFO_PRESENT flag is specified.

Reply:
Resolving as we know how it's done now